lib/tasks/templates.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Mix.Tasks.Addict.Gen.Templates do
+  use Mix.Task
+  import Mix.Generator
+  alias Addict.Utils
+
+  @shortdoc "Generate templates and view for Addict"
+  @moduledoc """
+  Generates an Addict view and templates
+      mix addict.gen.templates
```

Comment:
Ah, I might've explained myself in the wrong way. The collision I refer is one of "semantics"  and not in code collision. Since we ultimately generate both `templates` and `views` calling it `templates` seems a bit weird to me :smile: